validations.js

```diff
@@ -15,6 +17,6 @@ module.exports = {
             }),
             data: joi.object()
         }).or('notification', 'data'),
-        result: joi.array()
+        result: joi.any()
```

Comment:
Joi validations are used to generate swagger documentation and we should not use joi.any() in master